poc/tests/test_vdaf_ping_pong.py

```diff
@@ -185,35 +186,35 @@ def test_one_round(self) -> None:
         )
 
         agg_param = 23
-        (leader_state, msg) = vdaf.ping_pong_leader_init(
+        leader_state = vdaf.ping_pong_leader_init(
             verify_key,
             ctx,
             vdaf.encode_agg_param(agg_param),
             nonce,
             vdaf.test_vec_encode_public_share(public_share),
             vdaf.test_vec_encode_input_share(input_shares[0]),
         )
-        self.assertEqual(leader_state, Continued((0, measurement), 0))
+        assert isinstance(leader_state, Continued)
+        self.assertEqual(leader_state.prep_round, 0)
 
-        (helper_state, msg) = vdaf.ping_pong_helper_init(
+        helper_state = vdaf.ping_pong_helper_init(
             verify_key,
             ctx,
             vdaf.encode_agg_param(agg_param),
             nonce,
             vdaf.test_vec_encode_public_share(public_share),
             vdaf.test_vec_encode_input_share(input_shares[1]),
-            cast(bytes, msg),
+            cast(bytes, leader_state.outbound),  # XX cast needed?
```

Comment:
This indeed doesn't need a cast anymore, since we narrow `leader_state` to `Continued` above, and it is apparent from the constructor that the `outbound` attribute is of type `bytes`.